genesyscloud/resource_genesyscloud_responsemanagement_response.go

```diff
@@ -275,14 +275,24 @@ func createResponsemanagementResponse(ctx context.Context, d *schema.ResourceDat
 		sdkresponse.MessagingTemplate = buildSdkresponsemanagementresponseMessagingtemplate(messagingTemplate)
 	}
 
-	log.Printf("Creating Responsemanagement Response %s", name)
-	responsemanagementResponse, _, err := responseManagementApi.PostResponsemanagementResponses(sdkresponse, "")
-	if err != nil {
-		return diag.Errorf("Failed to create Responsemanagement Response %s: %s", name, err)
+	diagErr := WithRetries(ctx, 30*time.Second, func() *retry.RetryError {
+		log.Printf("Creating Responsemanagement Response %s", name)
+		responsemanagementResponse, resp, err := responseManagementApi.PostResponsemanagementResponses(sdkresponse, "")
+		if err != nil {
+			if IsStatus412(resp) {
```

Comment:
@dginty4 @charliecon Do you guys know who the API owners of Response Management.  PR looks good, but I am wondering why they are returning a 412 and whether it is appropriate to retry on a 412.